src/world/partner/bombette.c

```diff
@@ -385,25 +380,25 @@ ApiStatus func_802BD758_3184A8(Evt* evt, s32 isInitialCall) {
                 evt->functionTemp[1]--;
                 break;
             }
-
```

Comment:
All of the line spacing changes between here and 401 should ideally be discarded